app/models/user.rb

```diff
@@ -88,7 +74,7 @@ def website_domain
   end
 
   def is_moderator
-    is_global_moderator || community_user&.is_moderator || is_admin || false
+    is_global_moderator || community_user&.is_moderator || is_admin || community_user.privilege?('mod') || false
```

Comment:
What's the difference between is_moderator and privilege?('mod') ?

Reply:
is_moderator is the moderator status in the db (column on Community User), privilege?('mod') is the Moderator Ability. I try to connect those, because for compatibility with older code, it'd be quite an effort to remove the column.